pkg/controllers/management/rkeworkerupgrader/upgrade.go

```diff
@@ -260,17 +260,19 @@ func (uh *upgradeHandler) upgradeCluster(cluster *v3.Cluster, nodeName string) e
 	upgradeStrategy := cluster.Status.AppliedSpec.RancherKubernetesEngineConfig.UpgradeStrategy
 	toDrain := upgradeStrategy.Drain
 
-	toPrepareMap, toProcessMap, upgradedMap, notReadyMap, filtered, upgrading, done := uh.filterNodes(nodes, cluster.Status.NodeVersion, toDrain)
+	// get current upgrade status of nodes
+	status := uh.filterNodes(nodes, cluster.Status.NodeVersion, toDrain)
 
-	maxAllowed, err := CalculateMaxUnavailable(upgradeStrategy.MaxUnavailableWorker, filtered)
+	maxAllowed, err := CalculateMaxUnavailable(upgradeStrategy.MaxUnavailableWorker, status.filtered)
 	if err != nil {
 		return err
 	}
 
-	logrus.Debugf("cluster [%s] worker-upgrade: workerNodeInfo: nodes %v maxAllowed %v upgrading %v notReady %v "+
-		"toProcess %v toPrepare %v done %v", cluster.Name, filtered, maxAllowed, upgrading, len(notReadyMap), keys(toProcessMap), keys(toPrepareMap), keys(upgradedMap))
+	logrus.Infof("cluster [%s] worker-upgrade: workerNodeInfo: nodes %v maxAllowed %v upgrading %v notReady %v "+
```

Comment:
This can't be at Info, it's going to explode logs in a large cluster. Even then I would say this might be a bad log message, imagine 10k nodes in this message.

Reply:
Oh oops!! moved it to info just for testing, forgot changing back, thank you